cluster/gce/util.sh

```diff
@@ -2553,15 +2600,92 @@ function kube-up() {
     write-cluster-name
     write-controller-config
     create-autoscaler-config
-    create-master
-    create-nodes-firewall
-    create-nodes-template
-    if [[ "${KUBE_CREATE_NODES}" == "true" ]]; then
-      # Windows nodes take longer to boot and setup so create them first.
-      create-windows-nodes
-      create-linux-nodes
+    if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
+      echo "DBG: Generating shared CA certificates"
+      generate-shared-ca-cert
+      prepare-master
+      # the (shared) bearer token is needed for all TP/RP provisioning
+      export SHARED_APISERVER_TOKEN="$(secure_random 32)"
+      export KUBE_BEARER_TOKEN=${SHARED_APISERVER_TOKEN}
+      echo "DBG: shared bearer token: ${KUBE_BEARER_TOKEN}"
+      declare -a TPSERVER_RESERVED_IP
+      declare -a TPSERVER_RESERVED_INTERNAL_IP
+      declare -a TPSERVER_NAME
+
+      TPIP=${TPIP:-}
+      RPIP=${RPIP:-}
+
+      ### RP server need know what is tp's kubeconfig
+      TPCONFIG_METADATA=${TPCONFIG_METADATA:-}
+      prepare-tpmaster
+      prepare-rpmaster
+      export TPIP
+      export RPIP
+      echo "Start to create tp server"
+      for num in $(seq ${SCALEOUT_TP_COUNT:-1}); do
+        echo "Creating tpserver: ${TPSERVER_NAME[$num]:-}"   
+        export ARKTOS_SCALEOUT_SERVER_TYPE="tp"  
+        export CLUSTER_NAME="${INSTANCE_PREFIX}-tp-${num}"
+        export MASTER_NAME="${CLUSTER_NAME}-master"
+        export NODE_INSTANCE_PREFIX="${CLUSTER_NAME}-minion"
+        export WINDOWS_NODE_INSTANCE_PREFIX="${CLUSTER_NAME}-windows-node"
```

Comment:
we won't need this support on windows here.

Reply:
yes, we won't support windows node for scale-out, but default node template creation including windows and linux, we never remove windows section from node template creation, here is ensure created windows node template has correct name, even we didn't use them. otherwise, the script don't know what to be deleted when cluster-down. This should be removed once we delete all windows node template creation.